### PR TITLE
WIP - Update editors

### PR DESCRIPTION
**Pull request by jonoliver**
# TODO:
- [ ] Add useful packages
- [ ] Add example configs

### CODE REVIEW

**Comment by catheraaine** [comment]
See: #109 & #110

File: software/vs-code/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Visual Studio Code (VS Code)
+
+![VS Code Overview](./vs-code-overview.png)
+
+Visual Studio Code is a [popular](https://insights.stackoverflow.com/survey/2019?utm_source=Iterable&utm_medium=email&utm_campaign=dev-survey-2019#technology-_-most-popular-development-environments) and versatile code editor from Microsoft with support for a variety programming languages.
```
Comment (jonoliver) [inline software/vs-code/README.md]: 👏

File: software/emacs/README.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+[Emacs](https://www.gnu.org/software/emacs/) has existed from the dawn of time.
+
+![real_programmers_use_emacs](./real_programmers.png)
```
Comment (catheraaine) [inline software/emacs/README.md]: Is this link broken?

Comment (jordanleven) [reply software/emacs/README.md]: @catheraaine - Which link isn't working for you?

**Comment by jordanleven** [comment]
@catheraaine - The markdown link styles have been updated.

**Pull request closed without merging**